dice.toml

```diff
@@ -0,0 +1,29 @@
+[Server]
+Addr = '0.0.0.0'
+Port = 7379
+KeepAlive = 300
+Timeout = 300
+MaxConn = 0
+ShardCronFrequency = 1000000000
+MultiplexerPollTimeout = 100000000
+MaxClients = 200
+MaxMemory = 0
+EvictionPolicy = 'allkeys-lfu'
+EvictionRatio = 0.4
+KeysLimit = 20000
+AOFFile = './dice-master.aof'
+PersistenceEnabled = true
+WriteAOFOnCleanup = false
```

Comment:
These two seem contradictory?

Reply:
We already had both of these in config struct, so current generated config is just copy of default configs. Auto generated by `init-config` option. Let me remove the `PersistenceEnabled` flag doesn't seem to be used anywhere.
config/config.go
```
PersistenceEnabled     bool          `mapstructure:"persistenceenabled"`
WriteAOFOnCleanup      bool          `mapstructure:"writeaofoncleanup"`
```

Reply:
Let's just remove it as a part of this change if it's not used anywhere.